src/activity_handlers.cpp

```diff
@@ -811,10 +811,10 @@ void activity_handlers::game_do_turn( player_activity *act, player *p )
 
     //Deduct 1 battery charge for every minute spent playing
     if( calendar::once_every(MINUTES(1)) ) {
-        game_item.charges--;
+        game_item.ammo_consume( 1, p->pos() );
```

Comment:
Please factor these changes out a a separate PR as they are non-controversial and readily testable and can therefore be merged immediately and reduce the overall size of your PR